model_analyzer/result/constraint_manager.py

```diff
@@ -62,10 +62,11 @@ def check_constraints(constraints, run_config_measurement):
         """
 
         if constraints:
-            for model_metrics in run_config_measurement.data():
+            for (i, model_metrics) in enumerate(run_config_measurement.data()):
                 for metric in model_metrics:
-                    if type(metric).tag in constraints:
-                        constraint = constraints[type(metric).tag]
+                    if constraints[i] is not None and type(
```

Comment:
The function parameters list above states that constraints is a dict, but this seems to be a list?

Reply:
Updated parameters to indicate this is a list of dicts